src/paperless_tesseract/parsers.py

```diff
@@ -45,13 +45,23 @@ def get_thumbnail(self):
         """
 
         out_path = os.path.join(self.tempdir, "convert.png")
+        gs_out_path = os.path.join(self.tempdir, "gs_out.png")
 
         # Run convert to get a decent thumbnail
+
+        # https://github.com/danielquinn/paperless/issues/447
+        # call gs first
+        environment = os.environ.copy()
+        cmd = ["gs", "-q", "-sDEVICE=pngalpha",
+               "-o", gs_out_path, self.document_path]
+        if not subprocess.Popen(cmd, env=environment).wait() == 0:
```

Comment:
Is there a reason you need to pass `env`? As I read the [documentation](https://docs.python.org/3/library/subprocess.html?highlight=subprocess#subprocess.Popen), it's the default behaviour to inherit the current process' environment.

Reply:
I based this on the `Popen` call in `run_convert()`. But you are right, according to the docu it is probably unnecessary here. Would you prefer to have it without `env`?

Reply:
Yes, please. Any code that's not necessary takes concentration away from code that's relevant. You can also have a look at `run_unpaper()`, where `env` is not passed, as it's not needed.